File: java/Crud.java
import java.util.ArrayList;

// package java;

public class Crud {
  private ArrayList<Mahasiswa> list;// atribut list
  private ArrayList<String> listNim;// atribut list NIM

  Crud(){
    list = new ArrayList<Mahasiswa>();// memanggil objek list
    listNim = new ArrayList<String>();// memanggil objek listNim
  }

  public void tambah(Mahasiswa mhs){
    // method menambah class Mahasiswa ke list
    if(listNim.indexOf(mhs.getNim()) != -1){
      // error handle jika memiliki NIM yang sama
      System.out.println("data yang ingin anda tambah tidak unique value");
      return;
    }
    this.list.add(mhs);
    this.listNim.add(mhs.getNim());
  }

  public void tampil(){
    // menampilkan list dari Mahasiswa
    for(int i = 0;i < list.size();i++){
      System.out.println(
      (i + 1) 
      + "\nNIK : " + list.get(i).getNik()
      + "\nNIM : " + list.get(i).getNim()
      + "\nNama : " + list.get(i).getNama()
      + "\nJenis Kelamin : " + list.get(i).getJenisKelamin()
      + "\nAsal universitas : " + list.get(i).getAsalUniv()
      + "\nEmail Edu : " + list.get(i).getEmailEdu()
      + "\nProdi : " + list.get(i).getProdi()
      + "\nFakultas : " + list.get(i).getFakultas()
      + "\n"
      );
    }
  }

}

File: php/Crud.php
<?php
class Crud {
  private $list = array();// array untuk menyimpan kelas Mahasiswa
  private $listNim = array();// array untuk menyimpan nim dari mahasiswa

  public function __construct(){
    //
  }

  public function tambah($mhs){
    // menambahkan data ke array
    if(in_array($mhs->getNim(), $this->listNim)){// kalau key id dari data udah ada
      echo "data yang ingin anda tambah tidak unique value";
      return;
    }
    array_push($this->list, $mhs);
    array_push($this->listNim, $mhs->getNim());
  }

  public function tampil(){
    // menampilkan seluruh data pada array
    echo "<table>";
    echo "
    <tr>
      <th>No</th>
      <th>NIK</th>
      <th>NIM</th>
      <th>Nama</th>
      <th>Jenis Kelamin</th>
      <th>Asal Universitas</th>
      <th>Email Edu</th>
      <th>Prodi</th>
      <th>Fakultas</th>
      <th>Foto</th>
      </tr>
    ";
    for($i = 0;$i <count($this->list);$i++){
      echo
      "<tr> <td>".
      ($i + 1) 
      . " </td><td> " . $this->list[$i]->getNik()
      . " </td><td> " . $this->list[$i]->getNim()
      . " </td><td> " . $this->list[$i]->getNama()
      . " </td><td> " . $this->list[$i]->getJenisKelamin()
      . " </td><td> " . $this->list[$i]->getAsalUniv()
      . " </td><td> " . $this->list[$i]->getEmailEdu()
      . " </td><td> " . $this->list[$i]->getProdi()
      . " </td><td> " . $this->list[$i]->getFakultas()
      . "</td><td>" . "<img src=assets/" . $this->list[$i]->getFoto() . " alt=" .$this->list[$i]->getFoto() . ">" 
      . "</td></tr>"
      ;
    }
    echo "</table>";
  }

}

?>